.pytest_cache/README.md

```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
```

Comment:
This needs to be removed.

Reply:
Will update the PR sometime this weekend.